server/views/pages/appointments/create-and-edit/schedule.njk.test.ts

```diff
@@ -873,4 +880,157 @@ describe('Views - Create Appointment - Schedule', () => {
+          hour: 14,
+          minute: 30,
+          date: formatDate(tomorrow.setHours(14, 30), "yyyy-MM-dd'T'HH:mm:ss") as unknown as Date,
+        },
+        endTime: {
+          hour: 16,
+          minute: 0,
+          date: formatDate(tomorrow.setHours(16, 0), "yyyy-MM-dd'T'HH:mm:ss") as unknown as Date,
+        },
+      } as AppointmentJourney
+      viewContext.session.appointmentJourney.prisoners = [
+        { number: 'A1234BC', name: 'TEST01 PRISONER01', cellLocation: '1-1-1' },
+        { number: 'B2345CD', name: 'TEST02 PRISONER02', cellLocation: '1-1-2' },
+        { number: 'C3456DE', name: 'TEST03 PRISONER03', cellLocation: '1-1-3' },
+        { number: 'D4567EF', name: 'TEST04 PRISONER04', cellLocation: '1-1-4' },
+        { number: 'E5678FG', name: 'TEST05 PRISONER05', cellLocation: '1-1-5' },
+        { number: 'F6789GH', name: 'TEST06 PRISONER06', cellLocation: '1-1-6' },
+        { number: 'G7891HI', name: 'TEST07 PRISONER07', cellLocation: '1-1-7' },
+        { number: 'H8912IJ', name: 'TEST08 PRISONER08', cellLocation: '1-1-8' },
+        { number: 'I9123JK', name: 'TEST09 PRISONER09', cellLocation: '1-1-9' },
+        { number: 'J1234KL', name: 'TEST10 PRISONER10', cellLocation: '1-1-10' },
+      ]
+      viewContext.prisonerSchedules = viewContext.session.appointmentJourney.prisoners.map(prisoner => ({
+        prisoner,
+        scheduledEvents: getScheduledEventsForPrisoner(prisoner),
+      }))
+      viewContext.session.editAppointmentJourney = {} as EditAppointmentJourney
+    })
+
+    describe('Add prisoners', () => {
+      beforeEach(() => {
+        viewContext.session.editAppointmentJourney.addPrisoners = [
+          {
+            number: 'A1234BC',
+            name: 'TEST01 PRISONER01',
+            cellLocation: '1-1-1',
+          },
+        ]
+        viewContext.prisonerSchedules = viewContext.session.editAppointmentJourney.addPrisoners.map(prisoner => ({
+          prisoner,
+          scheduledEvents: getScheduledEventsForPrisoner(prisoner),
+        }))
+
+        $ = cheerio.load(compiledTemplate.render(viewContext))
+      })
+
+      it('should display "Attendees you are adding"', () => {
+        expect(getAppointmentDetailsValueElement('Attendees you are adding').text().trim()).toEqual('1')
+      })
+
+      it('should not display "Appointment name"', () => {
+        expect(getAppointmentDetailsValueElement('Appointment name').length).toEqual(0)
+      })
+
+      it('should display "Date" without change links', () => {
+        expect(getAppointmentDetailsValueElement('Date').text().trim()).toEqual(
+          formatDate(tomorrow, 'EEEE, d MMMM yyyy'),
+        )
+        expect($('[data-qa=change-start-date]').length).toEqual(0)
+      })
+
+      it('should display "Time" without change links', () => {
+        expect(getAppointmentDetailsValueElement('Time').text().trim()).toEqual('14:30 to 16:00')
+        expect($('[data-qa=change-time]').length).toEqual(0)
+      })
```

Comment:
I would add a `should display remove attendee links` to cover the logic on schedule.njk line 219. The equivalent test for create appointment is on line 440